src/components/pagination/_pagination.scss

```diff
@@ -130,6 +130,10 @@ $css--helpers: true;
       }
     }
 
+    .#{$prefix}--select-input:focus .#{$prefix}--select-option-label {
+      display: none;
```

Comment:
Is this change here to stable intentional...?

Reply:
No meant to be in experimental.